be/src/fs/fs_s3.cpp

```diff
@@ -177,6 +177,7 @@ S3ClientFactory::S3ClientPtr S3ClientFactory::new_client(const TCloudConfigurati
     if (!aws_cloud_credential.endpoint.empty()) {
         config.endpointOverride = aws_cloud_credential.endpoint;
     }
+    config.maxConnections = config::object_storage_max_connection;
```

Comment:
Does it have an effect on the data lake reader?

Reply:
It has no side effect on the data lake reader.

Reply:
how can we configure the concurrency for the data sync in share-data mode architecture?
cc @kevincai

Reply:
it is a different io framework for shared data, however most of the configuration items are similar, possibly expose similar configurations in that case.

Reply:
> possibly expose similar configurations in that case.

Are you going to cut a PR for that? We want to harness the most from the AWS network bandwidth and S3

Reply:
will take a look. :)